app/views/casework/crime_applications/_interests_of_justice.html.erb

```diff
@@ -29,15 +29,17 @@
   </table>
 <% else %>
   <dl class="govuk-summary-list govuk-!-margin-bottom-9">
-    <% crime_application.ioj_passport.each do |passport| %>
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">
-          <%= label_text(:justification) %>
-        </dt>
-        <dd class="govuk-summary-list__value">
-          <%= t(passport, scope: 'values') %>
-        </dd>
-      </div>
-    <% end %>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        <%= label_text(:justification) %>
+      </dt>
+      <dd class="govuk-summary-list__value">
+        <% if crime_application.ioj_passport.include?('on_age_under18') %>
```

Comment:
Could have a presenter for this if we think that this logic is a bit much for the views - though happy this way.

Reply:
Obviously change the method names to make more sense as you like :)